docs/content/get-started/set-up.md

```diff
@@ -1,9 +1,10 @@
 # Get started: 1 - Set up
-There are three parts to this step: 
+There are four parts to this step: 
```

Comment:
I love how the PR title is "Remove step" and yet we've ended up incrementing this 😂

Reply:
Part, Jamie, part. Not a step. Obvs very different.